woocommerce/pinterest-for-woocommerce
# Ads Credit Landing Page Update.

### Changes proposed in this Pull Request:

Update Landing Page Ads Credit section:
- Add missing graphics
- Update copy
- Add Terms And Conditions Modal with links to sections
- Add tracking to all of the clickable elements



Closes #535 .

### Screenshots:
<img width="864" alt="image" src="https://user-images.githubusercontent.com/17271089/191459353-007e5685-ae6e-4f58-9efb-3ba9e7c3d191.png">

<img width="864" alt="image" src="https://user-images.githubusercontent.com/17271089/191459618-1ea6d46d-969d-4809-a7ae-b14298cdce23.png">




### Detailed test instructions:


0. Be sure to build the plugin before testing.
1. Open Pinterest Tab ( make sure that you are not connected ).
2. Inspect the Ads Tab. Check that the copy matches.
3. Click on the terms and conditions link.
4. Inspect the modal. Check that the copy matches. Links work.

Inspect the documentation update.


### Changelog entry

> Add - Ads Credit Terms and Conditions Modal.
